src/lib/state.ts

```diff
@@ -81,21 +86,44 @@ interface ValueState extends TranslatedValueID {
 
 interface MetadataState {
```

Comment:
Should this be renamed to better distinguish it from `ConfigurationMetadataState` ?

Reply:
suggestions for what to call it? Not a bad idea, just don't know a better name 🤷🏾